OneLogin.xcodeproj/project.pbxproj

```diff
@@ -2121,6 +2145,8 @@
 				DEVELOPMENT_TEAM = N8W395F695;
 				GENERATE_INFOPLIST_FILE = YES;
 				INFOPLIST_FILE = "OneLogin-Info.plist";
+				INFOPLIST_KEY_CFBundleDisplayName = "One Login - Debug";
```

Comment:
I don't think these build settings should be here, are we able to move them into the `xcconfig` files instead?

Reply:
Yeah I don't get why Xcode seems to add these back intermittently, removed 🙂